articles/cookbook.xml

```diff
@@ -0,0 +1,90 @@
+            <p>     Mary E. Johnston, the artist and author of Recuerdos de Los Rancheros, seems to be forgotten in history. While there is hardly any information about her, the information acquired is significant. Johnston was a student at State Normal School of Los Angeles and belonged to the senior class of 1892 which consisted of 78 students (State Normal School Groups). As a matter of fact, she was one of the four female graduates of that class from Pasadena (History of Pasadena). Adding to this, State Normal School of Los Angeles was established in 1882 and served as a former name of UCLA until 1919. Technically, the author of this authentic cookbook was a graduate from UCLA campus before it became a University of California.</p>
+         </div>
+      </body>
+      
+      <back>
+         
+         <note type="mapmarker"><!-- this will be the title & text in your map marker pop-up -->
+            <placeName type="arts and culture">Recuerdos de Los Rancheros, San Pascual Stables</placeName>
+            <geo>-118.166599 34.123646</geo><!-- geocoordinates here. !! enter in the longitude first, then the lattitude !! you can look up coordinates here: http://www.gps-coordinates.net -->
+            <p>An original cookbook containing authentic hispanic recipes.  Thought to be one of the first cookbooks, it includes watercolor depictions of various Latin American foods.</p>
+         </note>
+         
+         <div type="bibliography">
+            <listBibl>
+               <bibl>Pilcher, Jeffrey M.. “Tamales or Timbales: Cuisine and the Formation of Mexican National Identity, 1821-1911”. The Americas 53.2 (1996): 193–216.</bibl>
+               <bibl>"California Ranchos." California Ranchos. Toucan Valley Publications, Inc., n.d.</bibl>
+               <bibl>Reid, Hiram Alvin, and Alfred James McClatchie. "History of Pasadena." Google Books.
+Pasadena History Company. (1895): 201.</bibl>
+               <bibl>"State Normal School Groups." (1892): 5-9.</bibl>
+            </listBibl>
+         </div>
+         
+         <div type="images">
+            <figure xml:id="fig1">
+               <graphic url="../images/Cookbook1.jpg"/>
+               <head></head>
+               <note type="attribution">San Pascuel: A Ranchero House, Image courtesy of UCLA Library Special Collections</note>
```

Comment:
Attribution should include the name of the creator (Mary Johnson) and the date (c1894), then "Image courtesy of...)